stable/gangway/values.yaml

```diff
@@ -9,6 +9,11 @@ image:
   pullSecrets: []
   # - name: secretName
 
+# Whether to use a root CA to trust for self signed certificates at the Oauth2 URLs.
+# If true, a file rootca.crt must be created containing the certificate (in format:
+# -----BEGIN CERTIFICATE----- ...)
+# trustedCAEnabled: false
```

Comment:
If you accept the proposed change to `configmap.yaml` this wording needs changing.

Reply:
Pls see comment above